### DISCUSSION TITLE
Running PSA Crypto API for STM32U5 with hardware accelerator

### DISCUSSION THREAD

**Discussion by Nano-HK** [zephyrproject-rtos/zephyr#74543] [Q&A]
Hi,

Is there a reference on how to run PSA API with hardware accelerator on STM32U5? I have been looking at the TF-M implementation and it looks like i need to do some CMake magic to integrate the backend. Is there a solution available already or do I need to do it myself?

Appreciate any help! :)

**Comment by ndrs-pst** [comment]
I think you might want to take a look at these links first:

- [Zephyr Project TFM Overview](https://docs.zephyrproject.org/3.6.0/services/tfm/overview.html)
- [Zephyr Project Trusted Firmware-M on GitHub](https://github.com/zephyrproject-rtos/trusted-firmware-m)

This topic is outside my area of expertise, but I hope this information helps point you in the right direction.
